IrConsoleApp/Lecture4/Lecture4Homework.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IrynaZharnasek.Lecture4
+{
+    public class Lecture4Homework
+    {
+        public static void Task1()
+        {
+            Console.WriteLine("Enter any positive number:");
+            int pNumb = Convert.ToInt32(Console.ReadLine());
+            int sum;
+            for (int i = 0; i < pNumb; i++)
+            {
+                sum = i + (i + 1);
+                Console.WriteLine($"The sum of numbers up to {pNumb} is:{sum}");
+            }
+        }
+
+        public static void Task2()
+        {
+            byte i = 0;
+            do
+            {
+                i++;
+                Console.WriteLine($"{i}*3={i * 3}");
+            }
+            while (i < 10);
+        }
+
+        public static void Task3()
```

Comment:
So you just multiplied all massive members manually, but what will you do if you have 1000 massive members?
tip: for/foreach loop can save your time

Reply:
Done